tasks/task_manager.hh

```diff
@@ -40,6 +40,13 @@ class messaging_service;
 
 namespace tasks {
 
+struct node_info {
+    gms::inet_address ip;
```

Comment:
Why do you need this?

Reply:
In case someone already used it. It preserves the compatibility with no additional cost. I think it's worth it

Reply:
Use it where? In the code? In this case the user should be changed as well. If you mean external user via the API then can we move id->ip translation to the API handler and use ids only internally?

Reply:
> If you mean external user via the API then can we move id->ip translation to the API handler and use ids only internally?
Yes, that's what I mean. Alright, I will move it